pkg/minikube/machine/cache_images.go

```diff
@@ -160,9 +160,6 @@ func replaceWinDriveLetterToVolumeName(s string) (string, error) {
 		return "", err
 	}
 	path := vname + s[3:]
-	if _, err := os.Stat(filepath.Dir(path)); err != nil {
```

Comment:
Unrelated change?

Reply:
Duh! You're quite right. I'd created the branch based on the commit which incorporated my other PR. Apologies. I've sorted that out and raised another PR (I could not figure out how to sort it out in the previous branch/PR) which can be found here https://github.com/kubernetes/minikube/pull/3262. Cheers.